fg-stitch-lib/src/align/aligners/mod.rs

```diff
@@ -257,20 +258,20 @@ impl Aligners<'_, MatchParams> {
                     self.opts.pre_align_min_score,
                 );
                 if let Some(score) = score_fwd {
-                    contig_idx_to_prealign_score.put(
-                        self.multi_contig
-                            .contig_index_for_strand(true, &target_seq.name)
-                            .unwrap(),
-                        score,
-                    );
+                    if let Some(idx) = self
+                        .multi_contig
+                        .contig_index_for_strand(true, &target_seq.name)
+                    {
+                        contig_idx_to_prealign_score.put(idx, score);
+                    }
                 }
                 if let Some(score) = score_revcomp {
-                    contig_idx_to_prealign_score.put(
-                        self.multi_contig
-                            .contig_index_for_strand(false, &target_seq.name)
-                            .unwrap(),
-                        score,
-                    );
+                    if let Some(idx) = self
+                        .multi_contig
+                        .contig_index_for_strand(false, &target_seq.name)
+                    {
+                        contig_idx_to_prealign_score.put(idx, score);
+                    }
```

Comment:
Question: The old implementation panics when `contig_index_for_strand` returns `None`, but this new implementation now _silently_ skips adding the score to the `contig_idx_to_prealign_score` `IndexMap`. Is this the desired behavior?